content/swiftui/concepts/views/terms/picker/picker.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Picker'
+Description: 'A control that selects a value from a set of exclusive values.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
```

Comment:
1. There should only be one line break after the metadata.
2. There should be a sentence describing a Picker before we reach the Syntax section.